data/json/monsters/zed_fusion.json

```diff
@@ -290,7 +290,7 @@
     "morale": 10,
     "vision_day": 60,
     "vision_night": 60,
-    "armor_bash": 50,
+    "armor_bash": 25,
```

Comment:
Was it intended to be included in current PR?

Reply:
Yes! 
Those monsters serve as walls too, reducing their armor is part of making the place easier to transverse